presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -124,6 +129,51 @@ public void testNormalBuildSql()
     }
 
     @Test
+    public void testBuildSqlWithString()
```

Comment:
This looks like a copy/paste of the above test. Can you add the string part to the above? Otherwise, get rid of all the non-string types so this test is easier to read (and doesn't duplicate the other one).